htdocs/js/pages/JobDetails.class.js

```diff
@@ -531,6 +531,7 @@ Class.subclass( Page.Base, "Page.JobDetails", {
 					],
 					backgroundColor: [
 						(cpu_avg < jcm*0.5) ? this.pie_colors.cool : 
+						(cpu_avg < jcm*0.5) ? this.pie_colors.cool :
```

Comment:
Thanks for implementing this, really appreciate it! But just as a question is this extra line a typo?